pkgs/development/python-modules/automx2/default.nix

```diff
@@ -30,6 +32,15 @@ buildPythonPackage rec {
     flask
     flask-migrate
     ldap3
+    pythonPackages.sdnotify
+  ];
+
+  patches = [
+    (fetchpatch {
+      name = "sdnotify.diff";
+      url = "https://github.com/eliasp/automx2/commit/32a6e038225716fd30c939d90973ef7e8aa7f5bd.diff";
```

Comment:
Please vendor the patch to easily update it

Reply:
Obsolete. See 2c23590dd6337b9b8db59ddcc56335f4311a430d